inventory-generation/tower-jobs-init/README.md

```diff
@@ -0,0 +1,14 @@
+# Ansible Tower
+
+This playbook generates an inventory based off of a list of `engagement_users` found in the `engagement.json` file. It parses out the users into the expected structure found [here](https://github.com/redhat-cop/infra-ansible).
+
```

Comment:
Can we please include some information about the source files it expects to find to successfully run? i.e.: the `email_templates_directory`, etc. etc.